v3/src/components/codap-modal.tsx

```diff
@@ -21,7 +21,7 @@ interface IProps<TContentProps> {
   onClose: () => void
   className?: string
   title: string
-  Icon?: React.FC<any>
+  Icon?: ReactNode
```

Comment:
The changes in this file do not appear to be used in this PR (or anywhere else as near as I can tell). I would recommend leaving them the way they were. I have in mind a refactoring of this code that would provide a better opportunity to make any such changes.